react/react dependency installer/8f8bd450-3f2b-45a5-abcc-12b633899c77.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": "react/React Dependency Installer",
+  "description": "Installs dependencies defined by the user for the react installer node. Node uses npm to install the dependencies on the same application path as the react application.",
+  "interface": {
+    "inputs": {
```

Comment:
We should point out in the descriptions that `dependencies` and `projectPath` are not protected from injection attack. (Or validate in code that they are actually path and package names)